### PR TITLE
Finish chap 8

### PR DESCRIPTION
**Pull request by Rhezar**

### CODE REVIEW

**Comment by Rhezar** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  def new
+  end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user && user.authenticate(params[:session][:password])
+      login user
+      redirect_to user
+    else
+      flash.now[:danger] = "Invalid email/password combination"
```
Comment (thangtx88) [inline app/controllers/sessions_controller.rb]: I18n

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def login(user)
```
Comment (thangtx88) [inline app/helpers/sessions_helper.rb]: bỏ ()

**Comment by thangtx88** [comment]
commented

**Comment by Rhezar** [comment]
ready

**Comment by thangtx88** [comment]
merged

**Pull request merged**